### PR TITLE
Add GitHub Docs

### PR DESCRIPTION
**Pull request by RehanSaeed**
@henrygab I've just been going through all my repos and adding all the GitHub docs, social media hero image etc. Hopefully, I haven't made copy paste errors. There is a new [funding.yml](https://help.github.com/en/articles/displaying-a-sponsor-button-in-your-repository) file which shows a 'sponsor us' button. For this you'd need to sign up to the GitHub beta and also share an email address so I can add you to opencollective.com.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# All contributions, however small are valued!
+
+# Steps to contribute
+
+ If you want to make If it's a small change, go ahead and raise a pull request, otherwise follow these steps:
```
Comment (henrygab) [inline .github/CONTRIBUTING.md]: Typo … line 5 here starts one sentence, then mid-way starts a different sentence.

File: .github/FUNDING.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+github: [RehanSaeed,henrygab]
```
Comment (henrygab) [inline .github/FUNDING.yml]: I would like to exclude myself from any funding.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (RehanSaeed) [reply .github/CONTRIBUTING.md]: Thank you for catching that copy pasta error!

File: .github/FUNDING.yml (hunk 2 above)
Comment (RehanSaeed) [reply .github/FUNDING.yml]: Done.

Comment (henrygab) [reply .github/FUNDING.yml]: Wouldn't that require removal of my alias from the FUNDING.yml?

**Comment by RehanSaeed** [comment]
Ah, sorry, forgot to push my changes.

**Pull request merged**